frontend/package.json

```
@@ -8,6 +8,9 @@
     "start": "next start",
     "lint": "next lint"
   },
+  "engines" : {
+    "node" : "20.10.0"
```

Vercel is complaining about this, can you change it to below? We'll be limited to version 18, but that's good enough until the end of 2025 :) 

```suggestion
    "node" : "18.x"
```